docs/integration/webapi.rst

```diff
@@ -128,7 +135,9 @@ You need to register the Autofac filter provider implementation because it does
 Implement the Filter Interface
 ------------------------------
 
-Instead of deriving from one of the existing Web API filter attributes your class implements the appropriate filter interface defined in the integration. The filter below is an action filter and  implements ``IAutofacActionFilter`` instead of ``System.Web.Http.Filters.IActionFilter``.
+Instead of deriving from one of the existing Web API filter attributes your class implements the 
```

Comment:
If there's a header for "Continuation Action Filter Interface" we should probably have one at the same level (`*****`) for something like "Standard Action Filter Interface." The first sentence here ("Instead of deriving...in the integration.") would stay under the main "Implement the filter interface" header, but then folks could see examples of the two action filters under their respective headers.

Reply:
I've added that heading and some extra words to preamble the standard interfrace.